irc/websocket.go

```diff
@@ -0,0 +1,56 @@
+package irc
+
+import (
+	"github.com/gorilla/websocket"
+	"net"
+	"net/http"
+	"time"
+)
+
+var upgrader = websocket.Upgrader{
+	ReadBufferSize:  1024,
+	WriteBufferSize: 1024,
+	CheckOrigin:     func(r *http.Request) bool { return true },
+}
+
+type WSContainer struct {
```

Comment:
Totally, I was not sure about that name and pushed before think to another name, my fault.